test/parameter_test.py

```diff
@@ -261,6 +275,16 @@ def test_enum_param_invalid(self):
     def test_enum_param_missing(self):
         self.assertRaises(ParameterException, lambda: luigi.parameter.EnumParameter())
 
+    def test_list_parameter(self):
```

Comment:
Good that you have tests. Can you also have a test where you test the command line parsing or reading from config?